aiida/workflows/arithmetic/add_multiply.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida-core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+"""Basic calcfunction-based workflows for demonstration purposes."""
+
+from aiida.engine import WorkChain, calcfunction, workfunction
+
+
+@calcfunction
+def add(x, y):  # pylint: disable=invalid-name
```

Comment:
why are these invalid names? In any case, I think it might be nicer to just add this as an exception to the `.pylintrc`. Add it to the `good-names` on line 139. For all the functions in this file that is.